test/network.bats

```diff
@@ -182,7 +182,7 @@ function teardown() {
 	# ensure that the server cleaned up sandbox networking if the sandbox
 	# failed after network setup
 	rm -f /var/lib/cni/networks/crionet_test_args/last_reserved_ip*
-	num_allocated=$(ls /var/lib/cni/networks/crionet_test_args | wc -l)
+	num_allocated=$(ls /var/lib/cni/networks/crionet_test_args | grep -v lock | wc -l)
```

Comment:
Ah, im wondering if we still need these changes...

Reply:
Yeah your PR will also need this i think

Reply:
Added it, thanks 🙏